cluster/gce/util.sh

```diff
@@ -660,7 +660,17 @@ function create-network() {
       --project "${PROJECT}" \
       --network "${NETWORK}" \
       --source-ranges "10.0.0.0/8" \
-      --allow "tcp:1-65535,udp:1-65535,icmp" &
+      --allow "tcp:1-2379,tcp:2382-65535,udp:1-65535,icmp" &
+  fi
+
+  # Create rule for accessing and securing etcd servers.
```

Comment:
I think this should be created in `create-master` method where we also create other firewall rules specific for the master.